test/gui/shared/scripts/names.py

```diff
@@ -125,3 +125,15 @@
 localDirectoryGroupBox_chooseLocalDirectoryButton_QToolButton = {"container": advancedConfigGroupBox_localDirectoryGroupBox_QGroupBox, "name": "chooseLocalDirectoryButton", "type": "QToolButton", "visible": 1}
 oCC_TlsErrorDialog_OCC_TlsErrorDialog = {"name": "OCC__TlsErrorDialog", "type": "OCC::TlsErrorDialog", "visible": 1}
 oCC_TlsErrorDialog_Yes_QPushButton = {"text": "Yes", "type": "QPushButton", "unnamed": 1, "visible": 1, "window": oCC_TlsErrorDialog_OCC_TlsErrorDialog}
+advancedConfigGroupBox_syncModeGroupBox_QGroupBox = {"container": contentWidget_advancedConfigGroupBox_QGroupBox, "name": "syncModeGroupBox", "type": "QGroupBox", "visible": 1}
+syncModeGroupBox_configureSyncManuallyRadioButton_QRadioButton = {"container": advancedConfigGroupBox_syncModeGroupBox_QGroupBox, "name": "configureSyncManuallyRadioButton", "type": "QRadioButton", "visible": 1}
+add_Folder_Sync_Connection_OCC_FolderWizard = {"type": "OCC::FolderWizard", "unnamed": 1, "visible": 1, "windowTitle": "Add Folder Sync Connection"}
+add_Folder_Sync_Connection_FolderWizardSourcePage_OCC_FolderWizardLocalPath = {"name": "FolderWizardSourcePage", "type": "OCC::FolderWizardLocalPath", "visible": 1, "window": add_Folder_Sync_Connection_OCC_FolderWizard}
+add_Folder_Sync_Connection_qt_passive_wizardbutton1_QPushButton = {"name": "__qt__passive_wizardbutton1", "type": "QPushButton", "visible": 1, "window": add_Folder_Sync_Connection_OCC_FolderWizard}
+add_Folder_Sync_Connection_groupBox_QGroupBox = {"name": "groupBox", "type": "QGroupBox", "visible": 1, "window": add_Folder_Sync_Connection_OCC_FolderWizard}
+groupBox_folderTreeWidget_QTreeWidget = {"container": add_Folder_Sync_Connection_groupBox_QGroupBox, "name": "folderTreeWidget", "type": "QTreeWidget", "visible": 1}
+add_Folder_Sync_Connection_qt_wizard_finish_QPushButton = {"name": "qt_wizard_finish", "type": "QPushButton", "visible": 1, "window": add_Folder_Sync_Connection_OCC_FolderWizard}
+add_Folder_Sync_Connection_Deselect_remote_folders_you_do_not_wish_to_synchronize_QLabel = {"text": "Deselect remote folders you do not wish to synchronize.", "type": "QLabel", "unnamed": 1, "visible": 1, "window": add_Folder_Sync_Connection_OCC_FolderWizard}
+add_Folder_Sync_Connection_Deselect_remote_folders_you_do_not_wish_to_synchronize_QTreeWidget = {"aboveWidget": add_Folder_Sync_Connection_Deselect_remote_folders_you_do_not_wish_to_synchronize_QLabel, "type": "QTreeWidget", "unnamed": 1, "visible": 1, "window": add_Folder_Sync_Connection_OCC_FolderWizard}
+deselect_remote_folders_you_do_not_wish_to_synchronize_QHeaderView_2 = {"container": add_Folder_Sync_Connection_Deselect_remote_folders_you_do_not_wish_to_synchronize_QTreeWidget, "orientation": 1, "type": "QHeaderView", "unnamed": 1, "visible": 1}
+size_HeaderViewItem = {"container": deselect_remote_folders_you_do_not_wish_to_synchronize_QHeaderView_2, "text": "Size", "type": "HeaderViewItem", "visible": True}
```

Comment:
Please verify that all these selectors are being used, and remove any unused elements. Also, if you find any previously added selectors that are not useful anymore, please remove them as well.